### DISCUSSION TITLE
Archiver tool question

### DISCUSSION THREAD

**Discussion by alexwillingham98** [thoughtspot/cs_tools#88] [Q&A]
Trying to run this command -- 
cs_tools tools archiver identify --content 'answer' 

Whenever I run this it tags answers and liveboards and not just answers. Am i missing something?

**Comment by alexwillingham98** [comment]
FYI i'm running on the latest release 1.4.13

**Reply by boonhapus** [reply]
Oh that's a cute bug. 😅 I'll squash it. Thanks @alexwillingham98 ! 

In the the meantime, if you're looking for just the list of answers you would tag, you can run...

`cs_tools tools archiver identify --content answer --syncer csv://directory=.`

... and then filter the CSV down by just the saved answers.

**Reply by alexwillingham98** [reply]
Thank you!
